standards/naming-things.md

```diff
@@ -0,0 +1,53 @@
+---
+category: Building software
+expires: 2018-03-15
+---
+## User needs
+
+Users should to be able to infer a basic understanding of what a given
+thing does from its name.
+
+## Principles
+
+Names should be self-descriptive – you should be able to infer a basic
+understanding of what something does from its name.
+
+Avoid naming things using puns, uncommon acronyms, version numbers, or
+with a brand. All of these make it difficult for others to understand
+what the thing does. Puns and acronyms (unless they're very widely
+understood outside government, like API) probably wont' be understood
+by anyone coming to your thing later. Version numbers and brand names
+(including names of parts of government) are likely to change over time
+as the thing adapts to changing user needs and policies.
+
+Ensure you use the same name consistently whenever you're referring to
+the same thing. For example, in most cases the name of an application's
+GitHub repository should match its hostname. Similarly, the team
+responsible for an application should probably be called something
+related to the application and its GitHub repositories.
+
```

Comment:
This should probably make explicit that names become part of your architecture, and unclear names make your architecture/systems unclear and harder to operate and maintain.